src/templates/ui-grid/uiGridHeaderCell.html

```diff
@@ -21,13 +21,13 @@
       aria-label="{{getSortDirectionAriaLabel()}}">
       <i
        ng-class="{ 'ui-grid-icon-up-dir': col.sort.direction == asc, 'ui-grid-icon-down-dir': col.sort.direction == desc, 'ui-grid-icon-blank': !col.sort.direction }"
-       title="{{isSortPriorityVisible() ? i18n.headerCell.priority + ' ' + col.sort.priority : null}}"
+       title="{{isSortPriorityVisible() ? i18n.headerCell.priority + ' ' + ( col.sort.priority + 1 )  : null}}"
```

Comment:
In Grid.js line 1880 aren't you bumping up the sortPriority, so adding 1 again here would be incorrect?